8.1.2/startup/initAppInsights.js

```diff
@@ -0,0 +1,8 @@
+try {
+    // Initialize the Application Insights runtime, using the
+    // instrumentation key provided by the environment.
+    require("applicationinsights").setup().start();
```

Comment:
Yep, doing this init step auto-registers a bunch of metric collection, in order to track telemetry without your app code needing to do anything at all.

If the user was already initializing the AI SDK, then the second call should effectively no-op, but let me verify that.